index.html

```diff
@@ -187,10 +183,42 @@ <h4>About myself</h4>
            external clients to reach their goals, mainly in the educational and 
            financial institutions sector. I can make your projects go ahead, 
            finish on time, improve and grow according to your evolving needs.
-
-        </p>
+          </p>
         <a href="#" class="superButtonR">Download Resume</a>
 
     </section>
+
+    <section id="contactForm">
+
+      <h4>CONTACT US</h4>
+      <p>If you have an application you are interested in developing, a feature that
+        you need built or a project that needs coding, I'd love to help with it.
+      </p>
+      
+      <div class="contact">
+        <form action="https://formspree.io/f/xrgjojpj" method="post">
+          <input type="text" id="name" name="user_name" placeholder="Name" required minlength="1" maxlength="30">
+          <input type="text" id="lastname" name="user_last_name" placeholder="Last name" required minlength="1" maxlength="30">
+          <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+          <textarea id="msg" name="user_message" placeholder="Your message" required minlength="1" maxlength="500"></textarea>
+          
+            <button class="superButton" type="submit">Send</button>
```

Comment:
- [ ] Please increase padding for the submit button and make sure it matches with the design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166520291-405f292a-7806-42b7-a4c9-91222cdc3a7e.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166520335-c4d99c09-156e-4363-a680-9093881f6a94.png)